modules/graphql_1/main.tf

```diff
@@ -151,6 +151,57 @@ resource "aws_lb_listener" "gql_app_lb_listener" {
   tags               = var.tags
 }
 
+# Network Load Balacner
+resource "aws_lb_target_group" "network_lb_tg" {
+  name        = "${random_id.lb_name.hex}-gql-n"
+  target_type = "alb"
```

Comment:
That isn't possible at the moment, the only target type options for Network Load Balancers are, instances, IP Addresses, and Application Load Balancers. This will have to be done when we do API gateway changes, when you create a VPC Link for REST API's you pick a Network Load Balancer to associate the created VPC Link with.

Reply:
So we will have a NLB pointing the target group to an ALB? I am not sure if/how the VPC link will work with the orca api gateway. Might need help from an architect who worked with vpc link before.

Reply:
Maybe add some notes on this ticket https://bugs.earthdata.nasa.gov/browse/ORCA-893

Reply:
Added notes to ticket